src/preset.js

```diff
@@ -44,4 +44,12 @@ module.exports = {
 
 	// Default is "node", but we need browser APIs
 	testEnvironment: 'jsdom',
+
+	// Load node build, not the browser build
+	testEnvironmentOptions: {
```

Comment:
```suggestion
	// Load node build, not the browser build
	// https://github.com/preactjs/preact/pull/3634#discussion_r930171882
	testEnvironmentOptions: {
```

Let's just add a small reference here for context but all good to go then!